frontend/src/metabase/static-viz/index.js

```diff
@@ -44,11 +48,48 @@ function getRawSeriesWithDashcardSettings(rawSeries, dashcardSettings) {
   });
 }
 
+function getVisualizerRawSeries(cards, dashcardSettings) {
+  const { columns, columnValuesMapping } = dashcardSettings.visualization;
+  const datasets = cards.reduce((acc, card) => {
+    if (card.card.id) {
+      acc[`card:${card.card.id}`] = card;
+    }
+    return acc;
+  }, {});
+
+  const dataSources = cards.map(card =>
+    createDataSource("card", card.card.id, card.card.name),
+  );
+
+  const mergedData = mergeVisualizerData({
+    columns,
+    columnValuesMapping,
+    datasets,
+    dataSources,
+  });
+
+  const { display, settings } = dashcardSettings.visualization;
+  return [
+    {
+      card: {
+        display,
+        visualization_settings: settings,
+      },
+      data: mergedData,
+      started_at: new Date().toISOString(),
```

Comment:
I think we can drop `started_at` from here, the FE relies on it for caching and we don't really need it for static viz